jenkins/deploy-clio-server.sh

```diff
@@ -170,59 +154,126 @@ render_ctmpls() {
   rm ${ctmpl_env_file}
 }
 
-stop_containers() {
-  local -r clio_fqdn=$1
+activate_service_account() {
+  VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+  SA_VAULT_PATH="secret/dsde/gotc/dev/common/ci-deployer-service-account.json"
+
+  docker run --rm -e VAULT_TOKEN="${VAULT_TOKEN}" \
+    broadinstitute/dsde-toolbox:dev vault read -format=json ${SA_VAULT_PATH} \
```

Comment:
I think `vault read -format=json ... | jq '.data'` is the same as `vault read -field=data ...`

Reply:
I think this may be true. Is there a reason to change it?

Reply:
I tried to use this and got the following error: "Field data not present in secret". It appears that it can get the fields inside of the json object's "data" field, but not the values of the json object itself, which includes metadata for the secret